sdk/include/sdk/vdb/IDataPointValueProvider.h

```diff
@@ -40,7 +40,7 @@ class IDataPointValueProvider {
     IDataPointValueProvider& operator=(const IDataPointValueProvider&) = delete;
     IDataPointValueProvider& operator=(IDataPointValueProvider&&)      = delete;
 
-    [[nodiscard]] virtual DataPointFailure getDataPointFailure() const = 0;
+    [[nodiscard]] virtual DataPointValue::Failure getDataPointValueFailure() const = 0;
```

Comment:
Simply call this `getFailure`? It already is a Value Provider for a single datapoint, hence the name can be short and concise.